lib/pdf_abr_writer.rb

```diff
@@ -6,7 +6,9 @@ class PdfAbrWriter
   include Lolrus
 
   attr_accessor :pdf_values
+  attr_accessor :pdf_cover_values
   attr_accessor :pdf_template_path
+  attr_accessor :pdf_template_with_cover_path
   attr_accessor :delivery_address
```

Comment:
can be removed if not used anymore

Reply:
The delivery address underneath (as above).  Yes.